### ISSUE TITLE
Error when output directory specified 

### ISSUE THREAD

**Issue by shernadi** [Oshlack/JAFFA#68]
Hi, 

I am experiencing some issues when I try to run JAFFAL with specifying an output directory (both when using `bpipe -d` or `-p jaffa_output=""` . It seems the pipeline fails when it try to generate the final output files. I am also wondering if there is an easy way to exclude the input file prefix from the output directory? 
```
Compiling the results from:
read.test
Error in file(file, "rt") : invalid 'description' argument
Calls: read.delim -> read.table -> file
Execution halted
ERROR: stage compile_all_results failed: Command in stage compile_all_results failed with exit status = 1 :
```

**Comment by nadiadavidson** [comment]
Thanks for bringing this to my attention. I'll look into a bug fix for this in the next release.

As a work around you can get JAFFAL to output into any directory:
1. Create and change into the directory where you want the output: 
mkdir MyOutputDir
cd MyOutputDir
2.  Run JAFFAL as normal (without the  -p jaffa_output="". It will default to the current directory).

The input file prefixes can't easily be removed from the output directories and names.

Please let me know if you have any further problem or questions.

**Comment by nadiadavidson** [comment]
Fixed in commit 5e809ade12b03f9a43a000c0b2317561d11b4db4 Along with #72 #70
